services/QuillLMS/config/routes.rb

```diff
@@ -313,6 +313,8 @@
     get 'progress_reports/landing_page' => 'progress_reports#landing_page'
     get 'progress_reports/activities_scores_by_classroom' => 'progress_reports#activities_scores_by_classroom'
     get 'progress_reports/real_time' => 'progress_reports#real_time'
+    get 'progress_reports/diagnostic_insights/*path' => 'progress_reports#diagnostic_insights'
```

Comment:
I think you can combine these both using an optional route parameter:
`get 'progress_reports/diagnostic_insights(/*path)' => 'progress_reports#diagnostic_insights'`